docs/sidebars.js

```diff
@@ -1,308 +1,321 @@
+            "building-assistants/default-actions",
+            "building-assistants/slot-validation-actions",
+          ],
         },
+        "building-assistants/llm-command-generator",
+        "building-assistants/responses",
         {
-            type: 'category',
-            label: 'Deploying Assistants',
-            collapsed: true,
-            items: [
-                'deploy/introduction',
-                'deploy/deploy-rasa',
-                'deploy/deploy-action-server',
-                'deploy/deploy-rasa-pro-services',
-            ],
+          type: "category",
+          label: "Config",
+          items: [
+            "building-assistants/llm-configuration",
+            "building-assistants/custom-graph-components",
+            "building-assistants/training-data-importers",
+            "building-assistants/graph-recipe",
+            "building-assistants/spaces",
+          ],
         },
+        "building-assistants/policies", // TODO: create-task
+        // TODO: create-task spit CLI docs
+        "building-assistants/command-line-interface",
+        // TODO: create-task add how-to-guides
+        "building-assistants/glossary",
         {
-            type: 'category',
-            label: 'Monitoring and Analyzing Assistants',
-            collapsed: true,
-            items: [
-                {
-                    type: 'category',
-                    label: 'Analytics',
-                    collapsed: true,
-                    items: [
-                        'monitoring/analytics/getting-started-with-analytics',
-                        'monitoring/analytics/realtime-markers',
-                        'monitoring/analytics/example-queries',
-                        'monitoring/analytics/data-structure-reference',
-                    ],
-                },
-                'monitoring/tracing',
-                'monitoring/load-testing-guidelines',
-            ],
+          type: "category",
+          label: "Building Classic Assistants",
+          collapsed: true,
+          items: [
+            // TODO: we'll need something for DM2 at some point
+            "building-classic-assistants/migrate-from",
+            // TODO: create-task spit CLI docs
+            "building-assistants/command-line-interface",
+            {
+              type: "category",
+              label: "Conversation Patterns",
+              collapsed: true,
+              items: [
+                "building-classic-assistants/chitchat-faqs",
+                "building-classic-assistants/business-logic",
+                "building-classic-assistants/fallback-handoff",
+                "building-classic-assistants/unexpected-input",
+                "building-classic-assistants/contextual-conversations",
+                "building-classic-assistants/reaching-out-to-user",
+              ],
+            },
+            {
+              type: "category",
+              label: "Config",
```

Comment:
I think this is a duplicate of what's in the top level nav, correct? IMO we should merge the two

Reply:
@amn41 not sure what you mean exactly? the idea is that DM1 and DM2 have 2 separate config documentation, because not everything is relevant in both modes.